### PR TITLE
Define "projection"; Fixes #221.

### LINKED ISSUE #221: Define "projection"
...and reference it from [projected].

### PR DESCRIPTION
**Pull request by CaseyCarter**
Despite being editorial, this is probably interesting enough to merit more eyes than just mine.

### CODE REVIEW

File: intro.tex [hunk 1]
```diff
@@ -83,6 +83,25 @@
 Terms defined in \cxxref{definitions} are used in this document with the same
 meaning.
 
+\def\definition{\definitionx{\section}}%
+
+\indexdefn{projection}%
+\definition{projection}{defns.projection}
+\defncontext{function object argument} which an algorithm uses to examine the
```
Comment (ericniebler) [inline intro.tex]: s/which/that/

**Comment by CaseyCarter** [comment]
"that" vs. "which" is the bane of my existence. Better?

**Comment by ericniebler** [comment]
> \defncontext{function object argument} used by an algorithm to examine the values of elements indirectly.

I don't love the use of the word "indirectly". So far, we've been using "indirectly" to mean "through a `Readable`". The projection doesn't get passed a `Readable`, it gets passed a value. I suggest striking "indirectly".

**Comment by ericniebler** [comment]
> "that" vs. "which" is the bane of my existence.

Try mentally excising the clause that begins with "which". If you can't do it without changing the meaning of the sentence, replace "which" with "that".

**Comment by CaseyCarter** [comment]
Killed "indirectly" in the definition of "projection".

**Pull request merged**